test/integration/targets/delegate_to/test_loop_control.yml

```diff
@@ -0,0 +1,16 @@
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: Test delegate_to with loop_control
+      ping:
+      delegate_to: "{{ item }}"
```

Comment:
Just wondering, this test doesn't specifically address the bug in #33981 
Is that okay?

Reply:
Yeah, the root cause is the same. Thanks for verifying the fix!